drivers/ethernet/eth_stm32_hal_v2.c

```diff
@@ -608,7 +652,7 @@ int eth_stm32_hal_init(const struct device *dev)
 	k_sem_init(&dev_data->tx_int_sem, 0, 1);
 
 	/* Tx config init: */
-	memset(&tx_config, 0, sizeof(ETH_TxPacketConfig));
+	memset(&tx_config, 0, sizeof(tx_config));
```

Comment:
can be removed https://github.com/zephyrproject-rtos/zephyr/pull/98218#discussion_r2460806481

Reply:
done